be/src/star_cache/admission_policy.h

```diff
@@ -0,0 +1,25 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#pragma once
+
+#include "star_cache/cache_item.h"
+
+namespace starrocks {
+
+enum class BlockAdmission : uint8_t {
+    FLUSH,
+    SKIP,
+    DELETE
+};
+
+class BlockKey;
+// The class to handle admission control logic
+class AdmissionPolicy {
+public:
+    virtual ~AdmissionPolicy() = default;
+
+    // Check the admisstion control for the given block
+    virtual BlockAdmission check_admission(const CacheItemPtr& cache_item, const BlockKey& block_key) = 0;
```

Comment:
Could you add some comments here to explain what's block key for? 

I can tell CacheItem & CacheId. But what's diff between CaheId & BlockKey? Why BlockKey here instead of CacheId?

Reply:
OK, I get it. BlockKey is the key of a Block in CacheItem.

Reply:
I'm still confused with what's the meaning of `Admission`? Admission to put block into memory block or ?